docs/user-guide.md

```diff
@@ -110,7 +181,57 @@ COMMANDS:
    help, h         Shows a list of commands or help for one command
 ```
 
-### Specifying single cluster
+### Checking best practices
+
+Use one of the following commands to check clusters against configuration best practices:
+
+* `./gke-policy check` followed by the cluster details or configuration file
+* `./gke-policy check best-practices` followed by the cluster details or configuration file
+
+The configuration best practices check validates GKE clusters against the set of
+GKE configuration policies.
+
+### Checking scalability limits
+
+Use `./gke-policy check scalability` followed by the cluster details or configuration file to check
+clusters against scalability limits.
+
+The scalability limits check validates GKE clusters against the GKE quotas and limits.
+The tool will report violations when the current values will cross the certain thresholds.
+
+**NOTE**: you need to run `kube-state-metrics` to export cluster metrics to use cluster scalability
+limits check. Refer to the [kube-state-metrics installation & configuration guide](#kube-state-metrics)
+for more details.
+
+#### Scalability check configuration
+
+  1. Ensure that `kube-state-metrics` is installed and configured on your cluster(s). Refer to the
+  [kube-state-metrics installation & configuration guide](#kube-state-metrics) for details
+  2. If Google Manged Service for Prometheus is used to collect metrics from `kube-state-metrics`,
+  ensure that IAM `roles/monitoring.viewer` role is in place. No other configuration is needed,
+  just run `./gke-policy check scalability` followed by your settings.
```

Comment:
we could add here an intermediate step with manually check in monitoring/prometheus console one of metrics used by our tool - to check if metrics themself are collected properly